recipe/build.sh

```diff
@@ -30,6 +30,7 @@ fi
 
 (bash configure --prefix=${PREFIX} \
                --host=${HOST} \
+               --with-libblosc=${PREFIX} \
```

Comment:
@xylar I don't know if it's related to the windows failure, but is this the right name? https://github.com/sthagen/OSGeo-gdal/blob/0faf9596670d78ee045f8be3f2c9f7840c30b969/configure.ac#L1690 seems to suggest it's `--with-blosc`

```suggestion
               --with-blosc=${PREFIX} \
```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=415135&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=877 (linux) has a warning:

```
configure: WARNING: unrecognized options: --with-libblosc
```

And the windows builds say that libblosc wasn't used. Maybe try this first, since the Windows error is consistent with blosc not being available.

Reply:
Oops, sorry about that!